app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminController < ApplicationController
```

Comment:
rename to `Admin::ApplicationController`

Reply:
Should I write Admin::ApplicationController instead of AdminController ?